src/shared/handle-api-response.js

```diff
@@ -6,6 +6,8 @@ import isEmpty from 'lodash/isEmpty'
 export default (response, route) => {
   // WP returns single objects or arrays
   const arrayResp = isArray(response)
+  const routes = idx(route, _ => _.config.routes)
+  let config = routes ? routes.filter(item => item.path === route.path) : null
```

Comment:
Perhaps `routeConfig` for this variable? Clearer we're dealing with current route being handled.